Parsers/ASimNetworkSession/Parsers/vimNetworkSessionSonicWallFirewall.yaml

```diff
@@ -0,0 +1,481 @@
+Parser:
+  Title: Network Session ASIM filtering parser for SonicWall firewalls

```

Comment:
I've provided comments for ASIM version, please apply the same to this parser as well.